README.Rmd

```diff
@@ -15,7 +15,7 @@ params:
 ===========
 
 <div class="subheader">
-Display simple diagnostic info of your Shiny app in top-right corner.
+Display simple diagnostic info of your Shiny app corner.
```

Comment:
missing "in" in here
could be:
"in its corner" 
"in one of its corners"
"in any of its corners"